README.md

```diff
@@ -345,6 +342,25 @@ analysis information from the hub.
       ```
       socketTimeoutMS: 300000,
       ```
+    * To specify a CodeSonar analysis ID of `<X>` which the CodeSonar plugin should use for evaluating analysis results, replace the `projectFile` line with the following.
+      ```
+      aid: <X>,
+      ```
+      For example, to specify an analysis ID of 81:
+      ```
+      aid: 81,
+      ```
```

Comment:
- move this point to last (it's the least likely I think?)
- change <X> to <AID> to be slightly more meaningful (and distinct from variables used in other bullets)
- add a note about why they might do this, and how usually they wouldn't

"To evaluate the results of a different analysis on your hub, replace the `projectFile` line with the following, where <AID> is the relevant *analysis id*`.
...
For example, to evaluate the results of the analysis with whose ID is 81:
...
In most cases the `projectFile` is the best way to identify the analysis of interest. However, you might want to use a different analysis if <reason>"